### PR TITLE
Upgrade branch 2.7.x using TemplateControl

### PR DESCRIPTION
**Pull request by octonato**
```
Updated with template-control on 2019-01-08T12:08:38.843Z
  /.travis.yml:
    wrote /.travis.yml
  **/build.properties:
    sbt.version=1.2.8

```

### CODE REVIEW

**Comment by dwijnand** [comment]
No, this removes changes and breaks the build.

**Comment by octonato** [comment]
This should not happen. We should be able to run template control without breaking the build.

**Comment by octonato** [comment]
I'm reopening it. I also remove the `merge-when-green` label to avoid it to be merged automatically. 

I want to review this and fix template control.

File: .travis.yml [hunk 1]
```diff
@@ -45,7 +45,6 @@ before_cache:
 
 cache:
   directories:
-    - "$HOME/.gradle/caches"
     - "$HOME/.ivy2/cache"
+    - "$HOME/.gradle/caches"
     - "$HOME/.jabba/jdk"
-    - "$HOME/.sbt/boot"
```
Comment (octonato) [inline .travis.yml:51]: @dwijnand, can you clarify why this line was added on a previous PR? Is that something that we should or want to add to template control and let it propagate to all samples?

Comment (dwijnand) [reply .travis.yml:51]: Yes

Comment (octonato) [reply .travis.yml:51]: I guess you are saying 'yes' to my second question, correct?
> Is that something that we should or want to add to template control and let it propagate to all samples?

Comment (dwijnand) [reply .travis.yml:51]: Yep!  The whole of `~/.sbt` should be cached in fact.

Comment (dwijnand) [reply .travis.yml:51]: I'll follow this up.

**Pull request merged**